airflow/configuration.py

```diff
@@ -495,6 +532,10 @@ def as_dict(
             set (True, default), or should the _cmd options be left as the
             command to run (False)
         :type include_cmds: bool
+        :param include_secret: Should the result of calling any *_secret config be
+            set (True, default), or should the _secret options be left as the
```

Comment:
I'm not sure this should be on by default... Hmmm

Reply:
`display_sensitive` is `False` by default, so by default it should show as `'< hidden >'`

Reply:
Oh right, that's alright then.